google/ads/google_ads/client.py

```diff
@@ -214,16 +113,34 @@ def load_from_storage(cls, path=None):
             IOError: If the configuration file can't be loaded.
             ValueError: If the configuration file lacks a required field.
         """
-        if path is None:
-            path = os.path.join(os.path.expanduser('~'), 'google-ads.yaml')
+        config_data = config.load_from_yaml_file(path)
+        kwargs = cls._get_client_kwargs(config_data)
+        return cls(**kwargs)
 
-        if not os.path.isabs(path):
-            path = os.path.expanduser(path)
+    @classmethod
+    def get_type(cls, name, version=_DEFAULT_VERSION):
+        """Returns the specified common, enum, error, or resource type.
 
-        with open(path, 'rb') as handle:
-            yaml_str = handle.read()
+        Args:
+            name: a str indicating the name of the type that is being retrieved;
+                e.g. you may specify "CampaignOperation" to retrieve a
+                CampaignOperation instance.
+            version: a str indicating the version of the Google Ads API to be
```

Comment:
Nit: remove "to be used" so it is cleaner and fits on one line

Reply:
Done